server.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"gopkg.in/juju/charm.v6-unstable"
```

Comment:
I'll have to have a conversation about this with Atom. The Go project plugin organizes these for me.